mesh_nvm.cpp

```diff
@@ -68,7 +69,9 @@ void mesh_nvm_initialize()
     }
 
     if (!mount_status) {
+        Nanostack::get_instance(); // ensure Nanostack is initialised
         ns_file_system_set_root_path("/fs/");
+        // Should be like: Nanostack::get_instance().file_system_set_root_path("/fs/");
```

Comment:
I don't understand what you are trying to say with this commit?
Proposal for a new API?

Reply:
You got it right, that kind of API should be added later on.